packages/types/src/documents/document.ts

```diff
@@ -39,6 +39,25 @@ export enum DocumentType {
   AUDIT_LOG = "al",
 }
 
+// these are the core documents that make up the data, design
+// and automation sections of an app. This excludes any internal
+// rows as we shouldn't import data.
+export const DocumentsToImport: DocumentType[] = [
```

Comment:
These are the "allowed to import doc types", right?

Reply:
Yeap, well less allowed and more desired - the ones we want, the rest we have no desire to import.